packages/xo-server/src/xo-mixins/rest-api.mjs

```diff
@@ -312,9 +318,30 @@ async function _getDashboardStats(app) {
     let skippedJobs = 0
     let successfulJobs = 0
     const backupJobIssues = []
+    const vmIdsProtected = new Set()
+    const vmIdsUnprotected = new Set()
 
     for (const job of jobs) {
-      if (!(await _jobHasAtLeastOneScheduleEnabled(job))) {
+      const jobHasAtLeastOneScheduleEnabled = await _jobHasAtLeastOneScheduleEnabled(job)
+
+      if (job.type === 'backup') {
+        const vmIds = extractIdsFromSimplePattern(job.vms)
+
+        for (const id of vmIds) {
+          if (vmIdsProtected.has(id)) {
```

Comment:
the vmId may not exists anymore, I think you should check if the ids is in the list ( and you already have a list of vm a little earlier when computing  nVms , so I would create a Map of all VMs uuuid=> { protected , hasOneEnabledJob ,.... } and use this as a base